buildroot-external/overlay/base/etc/init.d/S99SetupLEDs

```diff
@@ -51,6 +51,11 @@ start() {
     /usr/local/etc/rc.postlocal
   fi
   
+  # if we are in safemode we remove the safemode control-file
+  if [[ -e /etc/config/safemode ]]; then
```

Comment:
This if check is not necessary, rm will only remove stuff when it exists. Please revise.